lib/matplotlib/markers.py

```diff
@@ -213,12 +213,19 @@ def __init__(self, marker=None, fillstyle=None):
 
         Parameters
         ----------
-        marker : str or array-like, optional, default: None
+        marker : str, array-like, `~.path.Path`, or `~.markers.MarkerStyle`, \
+        default: None
             See the descriptions of possible markers in the module docstring.
 
         fillstyle : str, optional, default: 'full'
             'full', 'left", 'right', 'bottom', 'top', 'none'
+
+        normalize_path : bool, default: True
+            Whether to normalize a custom paths that are provided as array of
+            vertices or `~.path.Path`. If True, the marker is normalized to the
```

Comment:
I don't think this fully captures what we are doing as I would naively read this to imply we are filling the square or making the shape touch several of the edges.  Maybe 
"If True, the path is normalized to fit within a unit-square by affine scaling."?